nixos/roles/router/bird2/whq.conf

```diff
@@ -1,38 +1,58 @@
-protocol static uplink_whq_4 {
+# host config must include: PEER_IPV4, PEER_IPV6
+
+protocol static local_whq_4 {
   ipv4 {
     table master4;
   };
 
-  route 0.0.0.0/0 via 185.105.255.1%brfe onlink yes;
+  route 185.105.255.0/25 unreachable;
+  route 185.105.255.160/27 unreachable;
+  route 185.105.255.192/27 unreachable;
 }
 
-protocol static uplink_whq_6 {
+protocol static local_whq_6 {
   ipv6 {
     table master6;
   };
 
-  route ::/0 via 2a06:3a80:0400:ff01::1;
+  route 2a06:3a80::/40 unreachable;
 }
 
+
 protocol device {
   scan time 60;
 }
 
-protocol bfd {
-  strict bind on;
-  interface "ethtr";
+# bfd config in host-specific config in this location because the
+# parser does not allow using defines for interface names.
+#
+# protocol bfd {
```

Comment:
This is meant to be an explanation why something you'd expect here is only defined somehow in line 92? Or what is the reason for keeping this commented out part still in?